tooling/ssa_fuzzer/fuzzer/src/mutations/configuration.rs

```diff
@@ -179,46 +118,179 @@ pub(crate) const BASIC_INSTRUCTION_ARGUMENT_MUTATION_CONFIGURATION:
 /// Mutations of arguments of instructions
 #[derive(Copy, Clone, Debug)]
 pub(crate) enum ArgumentMutationOptions {
-    Random,
-    IncrementIndex,
-    DecrementIndex,
+    MutateIndex,
     ChangeType,
 }
 
-pub(crate) type ArgumentMutationConfig = WeightedSelectionConfig<ArgumentMutationOptions, 4>;
+pub(crate) type ArgumentMutationConfig = WeightedSelectionConfig<ArgumentMutationOptions, 2>;
 pub(crate) const BASIC_ARGUMENT_MUTATION_CONFIGURATION: ArgumentMutationConfig =
     ArgumentMutationConfig::new([
-        (ArgumentMutationOptions::Random, 1),
-        (ArgumentMutationOptions::IncrementIndex, 3),
-        (ArgumentMutationOptions::DecrementIndex, 3),
+        (ArgumentMutationOptions::MutateIndex, 7),
         (ArgumentMutationOptions::ChangeType, 2),
     ]);
 
-/// Mutations of value types
 #[derive(Copy, Clone, Debug)]
-pub(crate) enum ValueTypeMutationOptions {
+pub(crate) enum VecMutationOptions {
     Random,
+    Insertion,
+    Deletion,
+    Swap,
+    Mutation,
+    PushDefault,
 }
+pub(crate) type VecMutationConfig = WeightedSelectionConfig<VecMutationOptions, 6>;
+
+pub(crate) const BASIC_VEC_MUTATION_CONFIGURATION: VecMutationConfig = VecMutationConfig::new([
+    (VecMutationOptions::Random, 1),
+    (VecMutationOptions::Insertion, 7),
+    (VecMutationOptions::Deletion, 22),
+    (VecMutationOptions::Swap, 20),
+    (VecMutationOptions::Mutation, 100),
```

Comment:
Probably better to pick another name for this option. Mutation mutation doesn't explain what it does